akka-http-core/src/main/scala/akka/http/scaladsl/model/StatusCode.scala

```diff
@@ -142,6 +142,7 @@ object StatusCodes extends ObjectRegistry[Int, StatusCode] {
   val UnsupportedMediaType         = reg(c(415)("Unsupported Media Type", "The request entity has a media type which the server or resource does not support."))
   val RequestedRangeNotSatisfiable = reg(c(416)("Requested Range Not Satisfiable", "The client has asked for a portion of the file, but the server cannot supply that portion."))
   val ExpectationFailed            = reg(c(417)("Expectation Failed", "The server cannot meet the requirements of the Expect request-header field."))
+  val ImATeapot                    = reg(c(418)("I'm a teapot", "The resulting entity body MAY be short and stout."))
```

Comment:
If this is seriously meant to get in then the PR is missing Java API ;-)
You'd have to mirror it in `javadsl/model/StatusCodes`